ios/Video/RCTVideo.swift

```diff
@@ -241,6 +242,7 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
 
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
+        AudioSessionManager.shared.registerView(view: self)
```

Comment:
There is no unregisterView call, is it really expected ?

Reply:
Oh, I will add unregisterView to deinit 👍
(but this I guess would be fine as it is because there is week object list so it would deinit anyway)